doc/ug/getting_started/index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Getting Started"
+---
+
+Welcome!
+
+This guide will help you get OpenTitan up and running by instructing you how to:
+
+1. setup an adequate build/testing environment on your machine,
+2. clone the OpenTitan Git repository, and
+3. build OpenTitan software/hardware for the target of your choosing.
+
+![Getting Started Workflow](getting_started_workflow.svg)
+
+As shown in the diagram above, we currently support multiple build targets and workflows, including: Verilator, FPGA, and DV.
+However, if you are new to the project, we recommend simulation with Verilator, as this uses only free EDA tools, and does not require any additional hardware.
+
+## Environment Setup
+
+There are two choices for setting up an OpenTitan environment:
+1. \[Supported\] [manual installation]({{< relref "install_instructions" >}}), or
+2. \[Experimental\] [provided Docker container]({{< relref "/util/container/README.md" >}}).
+
+While the first option is the only one currently supported, the second option is the quickest to get running.
+If you plan on being a long term user of the project, and/or contributer, we recommend the first (manual installation) option.
+Alternatively, if your goal is to get a Verilator simulation up and running as quickly as possible, we recommend the second (container) option.
```

Comment:
Maybe `s/we recommend/you may find it easier to try/` or similar? I'm not sure we should recommend it that directly yet.